### ISSUE TITLE
can't import openmm, openmm app

### ISSUE THREAD

**Issue by mateoalpha98** [openmm/openmm#3748]
I have been using this code for the last 3 months:
from simtk.openmm.app import *
from simtk.openmm import *
from simtk.unit import * 
from simtk.openmm import app

it also worked without putting simtk.
but all of a sudden it doesn't work in my terminal. ¿is there another way to import these libraries?

this is the error message: 
Traceback (most recent call last):
  File "/home/mateoc/Documentos/practiques/tleap_propka/openmm.py", line 1, in <module>
    from openmm.app import *
  File "/home/mateoc/Documentos/practiques/tleap_propka/openmm.py", line 1, in <module>
    from openmm.app import *
ModuleNotFoundError: No module named 'openmm.app'; 'openmm' is not a package

thanks a lot!

**Comment by peastman** [comment]
That means you're using a Python environment that doesn't have OpenMM installed.  You might have multiple Python interpreters installed, or possibly multiple virtual environments.  You need to find the one you were using before.

**Comment by mateoalpha98** [comment]
> That means you're using a Python environment that doesn't have OpenMM installed. You might have multiple Python interpreters installed, or possibly multiple virtual environments. You need to find the one you were using before.
the environment i used for this is called openmm, so this is not the problem, i could also run it from base, and this problem also is happening to a mate.

**Comment by peastman** [comment]
What is the output of `conda list`?

**Comment by mateoalpha98** [comment]
# packages in environment at /home/mateoc/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             4.5                       1_gnu  
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asttokens                 2.0.5                    pypi_0    pypi
attrs                     21.4.0                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
black                     22.1.0                   pypi_0    pypi
bleach                    4.1.0                    pypi_0    pypi
brotlipy                  0.7.0           py39h27cfd23_1003  
ca-certificates           2022.6.15            ha878542_0    conda-forge
certifi                   2022.6.15        py39hf3d152e_0    conda-forge
cffi                      1.15.0           py39hd667e15_1  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
click                     8.0.3                    pypi_0    pypi
colorama                  0.4.4              pyhd3eb1b0_0  
conda                     4.13.0           py39hf3d152e_1    conda-forge
conda-package-handling    1.7.3            py39h27cfd23_1  
cryptography              36.0.0           py39h9ce1e76_0  
cudatoolkit               10.2.89              h8f6ccaa_8    conda-forge
debugpy                   1.5.1                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
distro                    1.7.0                    pypi_0    pypi
docutils                  0.17.1                   pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
executing                 0.8.2                    pypi_0    pypi
fftw                      3.3.10          nompi_h77c792f_102    conda-forge
idna                      3.3                pyhd3eb1b0_0  
ipykernel                 6.9.1                    pypi_0    pypi
ipython                   8.0.1                    pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                7.6.5                    pypi_0    pypi
jedi                      0.18.1                   pypi_0    pypi
jinja2                    3.0.3                    pypi_0    pypi
jsonschema                4.4.0                    pypi_0    pypi
jupyter                   1.0.0                    pypi_0    pypi
jupyter-client            7.1.2                    pypi_0    pypi
jupyter-console           6.4.0                    pypi_0    pypi
jupyter-core              4.9.2                    pypi_0    pypi
jupyterlab-pygments       0.1.2                    pypi_0    pypi
jupyterlab-widgets        1.0.2                    pypi_0    pypi
ld_impl_linux-64          2.35.1               h7274673_9  
libblas                   3.9.0           15_linux64_openblas    conda-forge
libcblas                  3.9.0           15_linux64_openblas    conda-forge
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0               h1234567_1  
libgfortran-ng            12.1.0              h69a702a_16    conda-forge
libgfortran5              12.1.0              hdcd56e2_16    conda-forge
libgomp                   11.2.0               h1234567_1  
liblapack                 3.9.0           15_linux64_openblas    conda-forge
libopenblas               0.3.20          pthreads_h78a6416_0    conda-forge
libstdcxx-ng              9.3.0               hd4cf53a_17  
markupsafe                2.0.1                    pypi_0    pypi
matplotlib-inline         0.1.3                    pypi_0    pypi
mistune                   0.8.4                    pypi_0    pypi
mmcif-pdbx                2.0.1                    pypi_0    pypi
mypy-extensions           0.4.3                    pypi_0    pypi
nbclient                  0.5.11                   pypi_0    pypi
nbconvert                 6.4.2                    pypi_0    pypi
nbformat                  5.1.3                    pypi_0    pypi
ncurses                   6.3                  h7f8727e_2  
nest-asyncio              1.5.4                    pypi_0    pypi
notebook                  6.4.8                    pypi_0    pypi
numpy                     1.23.0                   pypi_0    pypi
ocl-icd                   2.3.1                h7f98852_0    conda-forge
ocl-icd-system            1.0.0                         1    conda-forge
openmm                    7.7.0            py39h9717219_1    conda-forge
openssl                   1.1.1l               h7f98852_0    conda-forge
packaging                 21.3                     pypi_0    pypi
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pathspec                  0.9.0                    pypi_0    pypi
pdb2pqr                   3.5.2                    pypi_0    pypi
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pip                       21.2.4           py39h06a4308_0  
platformdirs              2.5.0                    pypi_0    pypi
prometheus-client         0.13.1                   pypi_0    pypi
prompt-toolkit            3.0.28                   pypi_0    pypi
propka                    3.4.0                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pycosat                   0.6.3            py39h27cfd23_0  
pycparser                 2.21               pyhd3eb1b0_0  
pygments                  2.11.2                   pypi_0    pypi
pyopenssl                 22.0.0             pyhd3eb1b0_0  
pyparsing                 3.0.7                    pypi_0    pypi
pyrsistent                0.18.1                   pypi_0    pypi
pysocks                   1.7.1            py39h06a4308_0  
python                    3.9.7                h12debd9_1  
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.9                      2_cp39    conda-forge
pyzmq                     22.3.0                   pypi_0    pypi
qtconsole                 5.2.2                    pypi_0    pypi
qtpy                      2.0.1                    pypi_0    pypi
readline                  8.1.2                h7f8727e_1  
requests                  2.27.1             pyhd3eb1b0_0  
ruamel_yaml               0.15.100         py39h27cfd23_0  
send2trash                1.8.0                    pypi_0    pypi
setuptools                58.0.4           py39h06a4308_0  
six                       1.16.0             pyhd3eb1b0_1  
sqlite                    3.38.0               hc218d9a_0  
stack-data                0.2.0                    pypi_0    pypi
terminado                 0.13.1                   pypi_0    pypi
testpath                  0.5.0                    pypi_0    pypi
tk                        8.6.11               h1ccaba5_0  
tomli                     2.0.1                    pypi_0    pypi
tornado                   6.1                      pypi_0    pypi
tqdm                      4.63.0             pyhd3eb1b0_0  
traitlets                 5.1.1                    pypi_0    pypi
typing-extensions         4.1.1                    pypi_0    pypi
tzdata                    2021e                hda174b7_0  
urllib3                   1.26.8             pyhd3eb1b0_0  
wcwidth                   0.2.5                    pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0  
widgetsnbextension        3.5.2                    pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7b6447c_0  
zlib                      1.2.11               h7f8727e_4  

to sum up, there's openmm version 7.7.0

**Comment by peastman** [comment]
What about this command?

```
python -c "import openmm"
```

**Comment by mateoalpha98** [comment]
from base:
(base) mateoc@mateoc-IdeaPad-5-15ITL05:~/Documentos/practiques/tleap_propka$ python -c "import openmm"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/mateoc/Documentos/practiques/tleap_propka/openmm.py", line 1, in <module>
    from openmm.app import *
ModuleNotFoundError: No module named 'openmm.app'; 'openmm' is not a package
(base) mateoc@mateoc-IdeaPad-5-15ITL05:~/Documentos/practiques/tleap_propka$ conda activate openmm

from openmm environment:
(openmm) mateoc@mateoc-IdeaPad-5-15ITL05:~/Documentos/practiques/tleap_propka$ python -c "import openmm"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/mateoc/Documentos/practiques/tleap_propka/openmm.py", line 1, in <module>
    from openmm.app import *
ModuleNotFoundError: No module named 'openmm.app'; 'openmm' is not a package

**Comment by peastman** [comment]
It looks like your environment has somehow gotten corrupted.  This happens a lot with conda, unfortunately.  Try creating a new environment with `conda create` and install OpenMM into it.  Does that work?

**Comment by mateoalpha98** [comment]
I tried yesterday but didn't work and it also installed a different version 7.4.0. which wasn't disired
if you have a code to install it properly may be it will help, thanks!

**Comment by raimis** [comment]
```
(openmm) mateoc@mateoc-IdeaPad-5-15ITL05:~/Documentos/practiques/tleap_propka$ python -c "import openmm"
Traceback (most recent call last):
File "", line 1, in
File "/home/mateoc/Documentos/practiques/tleap_propka/openmm.py", line 1, in
from openmm.app import *
ModuleNotFoundError: No module named 'openmm.app'; 'openmm' is not a package
```

It seems, you have `/home/mateoc/Documentos/practiques/tleap_propka/openmm.py`, which Python tries to import instead of the actual `openmm` package.

Try to run `python -c "import openmm"` in another directory or remove `/home/mateoc/Documentos/practiques/tleap_propka` from `PYTHON_PATH`.

**Comment by peastman** [comment]
Oh, good catch!  Right, this is an annoying thing with Python.  If your local directory contains a file or directory with the same name as an installed package, it overrides the package.
